phpunit.xml.dist

```diff
@@ -1,24 +1,12 @@
 <?xml version="1.0"?>
-<phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-    bootstrap="./vendor/autoload.php"
-    cacheResultFile="build/.phpunit.result.cache"
-    colors="true"
-    convertErrorsToExceptions="true"
-    convertNoticesToExceptions="true"
-    convertWarningsToExceptions="true"
-    verbose="true"
-    stopOnFailure="false"
-    processIsolation="false"
-    backupGlobals="false"
->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.3/phpunit.xsd" bootstrap="./vendor/autoload.php" colors="true" stopOnFailure="false" processIsolation="false" backupGlobals="false" cacheDirectory=".phpunit.cache">
```

Comment:
Can we keep the old formatting here? I find it easier to read with the previous layout.

Reply:
done

Reply:
Thanks; that helps. :)